wrappers/wasm/BarcodeReader.cpp

```diff
@@ -96,8 +97,21 @@ EMSCRIPTEN_BINDINGS(BarcodeReader)
 	        .field("format", &ReadResult::format)
 	        .field("text", &ReadResult::text)
 	        .field("error", &ReadResult::error)
+            .field("position", &ReadResult::position)
 	        ;
 
+    value_object<ZXing::PointI>("Point")
+	        .field("x", &ZXing::PointI::x)
+	        .field("y", &ZXing::PointI::y)
+	        ;
+
+    value_object<ZXing::Position>("Position")
+            .field("topLeft", emscripten::index<0>())
```

Comment:
emscripten complains if I set this to `.field("topLeft", ZXing::Position::topLeft)`
```
error: pointer to function type 'emscripten::internal::GenericBindingType<ZXing::PointT<int> () const noexcept>::ActualT' (aka 'ZXing::PointT<int> () const noexcept') cannot have 'const' qualifier
            typedef ActualT* WireType;
                           ^
```

Reply:
`ZXing::Position::topLeft` is a member function. I guess you need to specify that differently.